libs/blocks/ost/ost.js

```diff
@@ -0,0 +1,10 @@
+import { createTag } from '../../utils/utils.js';
+import { initOst } from '../../deps/ost-wrapper.js';
+
+export default async function init(el) {
+  const ostRootContainer = document.querySelector('.ost');
+  ostRootContainer.removeChild(ostRootContainer.firstElementChild);
+  const ostRoot = createTag('div', { id: 'root' });
```

Comment:
there might be other div element with `id="root"`
Actually the container doesn't need an id as long as you keep the dom element reference.

Reply:
This code should only be called on ost page so there shouldn't be another div with `id=root`. I tried removing it and testing and it didn't work, my guess is React doesn't play well with milo semantic. Any suggestion @auniverseaway ?

Reply:
This is your own internal tooling causing this issue: https://git.corp.adobe.com/Dexter/tacocat.js/blob/MWPW-121450/packages/offer-selector-tool/src/global.jsx#L147.

I would probably make that querySelector class based instead of ID.

Reply:
Done, now using the block div with class `ost` instead ✅